AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1487,7 +1487,7 @@ runningnews.gr#%#//scriptlet("remove-attr", "oncontextmenu", "[oncontextmenu]")
 runningnews.gr#%#//scriptlet("remove-attr", "ondragstart", "[ondragstart]")
 runningnews.gr#%#//scriptlet("remove-attr", "onselectstart", "[onselectstart]")
 runningnews.gr#%#//scriptlet("remove-attr", "onmousedown", "img[onmousedown]")
-tistory.com,runningnews.gr#%#//scriptlet('set-constant', 'window.disableselect', 'noopFunc')
+janndk.com,tistory.com,runningnews.gr#%#//scriptlet('set-constant', 'window.disableselect', 'noopFunc')
```

Comment:
I don't need the rule on my end. Please check this.

Reply:
They are removed. https://github.com/AdguardTeam/AdguardFilters/pull/91173/commits/1264596965bbab090b5acd338bb52920de4d9946